### PR TITLE
[MB-2052] Release 3.2.0

### PR DESCRIPTION
**Pull request by hcrowell**
Unity Release 3.2.0
* Update Urban Airship Android SDK to 8.2.0
* Update Urban Airship iOS SDK to 8.1.3
  The `UANativeBridge.js` file may need to be manually deleted in `Assets/Plugins/iOS/Airship/AirshipResources.bundle`.

Testing:
* Received push notifications on iOS 10.1.1 iPod and Android 7.0 Nexus5x

### CODE REVIEW

**Comment by rlepinski** [comment]
👍

**Comment by crow** [comment]
🍔 lgtm

**Comment by hcrowell** [comment]
ToDo: Update to latest Android 8.2.1 release (scheduled for today) before merging into next

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,12 @@
 UA Unity ChangeLog
 ==================
 
+Version 3.2.0 - December 5, 2016
+================================
+ - Updated Urban Airship iOS Library to 8.1.3
+   The `UANativeBridge.js` file may need to be manually deleted in `Assets/Plugins/iOS/Airship/AirshipResources.bundle`.
```
Comment (rlepinski) [inline CHANGELOG]: You can remove this. js file was introduced in 8.1.0. The last version of unity is 8.0.2

Comment (hcrowell) [inline CHANGELOG]: Thanks Ryan! Good catch!

**Pull request merged**